webapp/src/pages/Settings.tsx

```diff
@@ -179,15 +174,16 @@ const NumberField: React.FC<
 
 const Settings: React.FC = () => {
   const { jobId } = useParams<{ jobId: string }>();
+  const { data: defaultConfig } = getDefaultConfigEndpoint.useQuery({ jobId });
   const { data: config } = getConfigEndpoint.useQuery({ jobId });
   const [updateConfig] = updateConfigEndpoint.useMutation();
 
   const [partialConfig, setPartialConfig] = React.useState<
     Partial<AzimuthConfig>
   >({});
 
-  // If config was undefined, PipelineCheck would not even render the page.
-  if (config === undefined) return null;
+  // If config or defaultConfig was undefined, PipelineCheck would not even render the page.
+  if (config === undefined || defaultConfig === undefined) return null;
```

Comment:
That's not true for `defaultConfig`. We should revert that change and handle loading/errors with `defaultConfig`. We'll need `isLoading` and `error` from
```diff
-  const { data: defaultConfig } = getDefaultConfigEndpoint.useQuery({ jobId });
+  const { data: defaultConfig, isLoading, error } = getDefaultConfigEndpoint.useQuery({ jobId });
```
and
```suggestion
  // If config was undefined, PipelineCheck would not even render the page.
  if (config === undefined) return null;

  if (isLoading) {
    return <Loading />;
  } else if (error || defaultConfig === undefined) {
    return (
      <Box alignItems="center" display="grid" justifyItems="center">
        <img src={noData} width="50%" alt="No confusion matrix data" />
        <Typography>{error?.message || UNKNOWN_ERROR}</Typography>
      </Box>
    );
  }
```

Reply:
Okay, Yes, I agree with the importance of it. Thanks

Reply:
Don't forget to revert the comment as I suggested.

Reply:
Okay, Sorry I missed out that part. I will add it.Thanks